vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java

```diff
@@ -418,28 +418,131 @@ public void testImportDupPolicyLegacy() throws RepositoryException, IOException,
         assertNull(duplicateNode);
     }
 
+    // constants for behavior target state to be tested
+
+    private static enum TS {
+        CONFLICT_TARGET_MOVED, CONFLICT_TARGET_PRESENT, NO_CONFLICT_TARGET_GONE, NO_CONFLICT_TARGET_UNCHANGED
+    };
+
+    // make boolean expectations readable
+
+    private static Boolean ID_NEW = true;
```

Comment:
why not encapsulating the semantics within the `enum TS`?

Reply:
too many variants...?